tsdb/engine/b1/b1.go

```diff
@@ -549,32 +549,37 @@ type Tx struct {
 	engine *Engine
 }
 
-// Cursor returns an iterator for a key.
-func (tx *Tx) Cursor(key string, direction tsdb.Direction) tsdb.Cursor {
-	// Retrieve key bucket.
-	b := tx.Bucket([]byte(key))
+// Cursor returns an iterator for a key over a single field.
```

Comment:
The `Cursor()` was to do a single field originally. It was too hard to split that out right now so we'll have to wrap the `pd` storage with a wrapping cursor to join individual field cursors.